### DISCUSSION TITLE
How to integrate HSM in community edition.

### DISCUSSION THREAD

**Discussion by sysadminbioenable** [Keyfactor/ejbca-ce#675] [Q&A]
i am trying to integrate hsm with ejbca in community edition. will it possible to do that?
 I have deployed  EJBCA Community Edition using Docker (keyfactor/ejbca-ce) image

**Comment by primetomas** [comment]
You can find documentation here:
https://docs.keyfactor.com/ejbca/latest/hardware-security-modules-hsm

Also see this GitHub repo that have HSM integration examples for the container:
https://github.com/Keyfactor/keyfactorcommunity
